pkgs/development/python-modules/kafka-python/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, buildPythonPackage, fetchPypi, tox }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  version = "1.3.5";
+  pname = "kafka-python";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "19m9fdckxqngrgh0www7g8rgi7z0kq13wkhcqy1r8aa4sxad0f5j";
+  };
+
+  buildInputs = [ tox ];
+
+  meta = with stdenv.lib; {
+    description = "Pure Python client for Apache Kafka";
+    homepage = https://github.com/dpkp/kafka-python;
+    license = licenses.asl20;
+    maintainers = with maintainers; [ ];
```

Comment:
Will you maintain this?

Reply:
I can try, but there will be no guarantee that I'll do it reliably :)

Reply:
If you can add an expression you can maintain it. It basically means you become responsible for the functioning. In principle we shouldn't have any (new) expressions without maintainers.

Reply:
Sure, I can fix it when it breaks.